docs/index.rst

```diff
@@ -38,6 +38,7 @@ Discover also `eXo UX Guidelines <http://exoplatform.github.io/ux-guidelines/>`_
    GettingStarted
    PersonalApplications
    ManageTasks
+   News
```

Comment:
Shouldn't this chapter go inside another one, like Manage-Space ?

Reply:
The news will be available from home later. 
it is not a feature that is made primarily for spaces, it's only temporary. That's why i created an inidividual chapter.

Reply:
Yes but that seems too high in the level hierarchy. We probably need some chapter about publishing content, News being one of the way to do it.

Reply:
https://docs.exoplatform.org/en/latest/GettingStarted.html#using-the-activity-stream
from here (as we share the news from composer) ?

Reply:
Indeed it seems to be a better place :+1: , we can add a new chapter in https://docs.exoplatform.org/en/latest/GettingStarted.html#share-as